pkg/sdk/views_gen.go

```diff
@@ -189,34 +213,31 @@ type DescribeViewOptions struct {
 	view     bool                   `ddl:"static" sql:"VIEW"`
 	name     SchemaObjectIdentifier `ddl:"identifier"`
 }
-
-// TODO [SNOW-965322]: extract common type for describe
-// viewDetailsRow is a copy of externalTableColumnDetailsRow.
 type viewDetailsRow struct {
-	Name       string         `db:"name"`
-	Type       DataType       `db:"type"`
-	Kind       string         `db:"kind"`
-	IsNullable string         `db:"null?"`
-	Default    sql.NullString `db:"default"`
-	IsPrimary  string         `db:"primary key"`
-	IsUnique   string         `db:"unique key"`
-	Check      sql.NullString `db:"check"`
-	Expression sql.NullString `db:"expression"`
-	Comment    sql.NullString `db:"comment"`
-	PolicyName sql.NullString `db:"policy name"`
+	Name          string         `db:"name"`
+	Type          DataType       `db:"type"`
+	Kind          string         `db:"kind"`
+	Null          string         `db:"null?"`
```

Comment:
why tehese fields are renamed (IsNullable, IsPrimary, IsUnique)? Is this the gen override?

Reply:
Yes. Now they're matching names in Snowflake.

Reply:
The problem with the names here is that the snowflake ones are less descriptive (and IsXxx was helpful). But okay, let's nor revert them